docs/book/installation/sylius_plus_installation.rst

```diff
@@ -1,384 +1,28 @@
-
 .. rst-class:: plus-doc
 
 Sylius Plus Installation
 ========================
```

Comment:
```suggestion
Sylius Plus
========================
```
I'm just wondering if this filename and topic are still misleading after these changes and if they should be renamed to something more general

Reply:
This is not a doc telling you what is Sylius Plus but how to install it :D even though it does not contain specific steps anymore.